Svc/FileManager/test/int/test_cmd_fileManager.py

```diff
@@ -0,0 +1,47 @@
+""" test_cmd_FileManager.py:
+
+Test the command FileManager with basic integration tests.
+    fileManager.CreateDirectory
+    fileManager.ShellCommand
+    fileManager.FileSize
+    fileManager.MoveFile
+    fileManager.AppendFile
+    fileManager.RemoveFile
+    fileManager.RemoveDirectory
+
+"""
+
+
+def test_send_command(fprime_test_api):
+    """Test that commands may be sent
+
+    Tests command send, dispatch, and receipt using send_and_assert command with a pair of FileManager commands.
+    """
+
+    deployment = fprime_test_api.get_deployment()
+
+    fprime_test_api.send_and_assert_command(deployment + "." + "fileManager.CreateDirectory", ["/tmp/file"], max_delay=1)
```

Comment:
This file path should not be hardcoded. I think we need to discuss with @LeStarch about how to best handle these test files and directories. I'm mainly thinking of cases where the file system has a specific file structure requirement e.g., microfs.